cmd/directpv/run.go

```diff
@@ -185,6 +185,10 @@ func run(ctx context.Context, args []string) error {
 				return fmt.Errorf("error while initializing drive discovery: %v", err)
 			}
 			klog.V(3).Infof("Drive discovery finished")
+		} else {
+			if err := node.SyncDrives(ctx, identity, nodeID, rack, zone, region, loopbackOnly); err != nil {
```

Comment:
This doesn't make sense when dynamic drive discovery runs as separate container. The container does sync first and waits/processes uevents.

Reply:
when dynamic discovery runs as a separate container, it runs alongside with the node server container. So, node server will be running parallel with dynamic-discovery container. 

This may lead to staging and publish volume calls to happen event before the first sync of the drives.

@balamurugana